components/sup/src/manager/mod.rs

```diff
@@ -406,9 +406,9 @@ impl Manager {
         Self::new(cfg, fs_cfg, launcher)
     }
 
+    #[cfg_attr(windows, allow(unused_variables))]
```

Comment:
Ah, I have a fix for this in the code I'm working on now... but I think I may like this fix better 😅